src/components/binding/binding-panel.js

```diff
@@ -218,7 +219,7 @@ genotet.BindingPanel.prototype.updateTracks = function() {
     if (!ui.length) {
       this.addTrack_();
     }
-    this.selectGenes_[index].val(track.fileName).trigger('change');
+    this.selectGenes_[index].val(track.gene).trigger('change');
```

Comment:
fileNames are keys while genes are not. If you want to use keys for binding data, fileNames are your choice. However the user recognizes a data file by both filename and descriptive gene name (data name). Right now we didn't implemented the correct UI for showing the list. The UI should show both the data name and the file name.